pandas/tests/frame/indexing/test_mask.py

```diff
@@ -0,0 +1,83 @@
+"""
+Tests for DataFrame.mask; tests DataFrame.where as a side-effect.
```

Comment:
why is this not tests/frame/methods/test_mask and tests/frame/methods/test_where?

Reply:
where/mask are indexing methods

Reply:
I don't understand the criteria for this directory, test_at_time, test_asof, test_between_time and other similar methods are in tests/frame/methods/test_*

Reply:
thats a fair question.  ill add a docstring to the directory `__init__` to try to clarify